### PR TITLE
HTML, CSS & JS capstone project.

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
This is a website for the Coding Outdoors Community (COC), there are 2 pages, home, about pages. The website is responsive to 768px.
- HTML & CSS for the two pages
- Create organizers page dynamically with JavaScript (Store data)
- Record loom video and present the capstone project

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                        <li class="nav-item">
+                            <a href="index.html" data-after="Home">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="about.html" data-after="About">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="./ExplorePage/index.html" data-after="Tieckts">Tickets</a>
+                        </li>
+                    </ul>
```
Comment (henatan99) [inline about.html:55]: - [x] Please, switch the `nav-list` and `logo` class components. The hamburger button is on the left in the original design. 
- [x] The nav section in the mobile view should not have any text. In addition, in the home page it has some background color. Please avoid the text and try to make the background color closely similar to the original design. You can see the screenshot images for comparison. :pray: 

image        
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+        type="text/css" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./css/main.css">
+    <title>CODE&OUTDOORS</title>
+</head>
+
+<body>
+    <!-- Header -->
+    <nav id="header">
```
Comment (henatan99) [inline index.html:31]: - [x] The nav section in the desktop is divided into a top dark bar and a bottom white bar. You did not include the top bar. Kindly add it to the nav section.
- [x] The logo and the link elements should be aligned in a row for the desktop view (min-width 768px). But for some pixel sizes above min-width, the elements are wrapped down to the next row. Kindly compare using the attached screenshot and make the required changes. 
- [x] Contents are flowing beyond the screen size in the desktop view. Kindly fix this. 

image       
image

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,747 @@
...
+
+/* End Partner section */
+
+/*  Footer section */
+
+#footer {
+  background-color: var(--secondary-color);
+  height: 25vh;
+  padding: 20px;
+}
```
Comment (henatan99) [inline css/main.css]: - [x] The footer section background color for the mobile view is white as in the original design. Please make changes accordingly. 
- [x] The copyright text in your design is not aligned properly. Kindly fix this. :pray: 

image        
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    <!-- End of Header -->
+    <!--     hero section -->
+    <section id="hero">
+        <div class="hero container d-flex flex-dir-col">
+            <div class="hero-title">
+                <h2>Coding <span></span></h2>
+                <h2>& Travelling <span></span></h2>
+                <h2>Meeting <span></span></h2>
+            </div>
+            <div class="content-home">
```
Comment (henatan99) [inline index.html]: - [x] There should be a text before the title in this section.

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,747 @@
...
+  align-items: center;
+  cursor: pointer;
+  transform: scale(0.8);
+  margin-right: 5px;
+}
+
+/* Hero Section  */
+#hero .container {
+  height: 70vh;
+}
```
Comment (henatan99) [inline css/main.css:200]: - [x] Please compare paddings, typography, and colors in your hero container with the original design. 
- [x] There are some texts which are not properly displayed.

image
image

File: css/main.css [hunk 6]
```diff
@@ -0,0 +1,747 @@
...
+  background-image: url("../Images/15.jpeg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top center;
+  position: relative;
+  z-index: 1;
+  width: 100vw;
+  text-align: center;
+  color: var(--white);
+}
```
Comment (henatan99) [inline css/main.css]: - [x] In the home page, main program section there is a problem of elements alignment and padding for the desktop view (min-width 768px)
- [x] As you can see, there are overflowing texts in the article elements. Kindly fix these issues. 

image
image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+
+            </article>
+            <article class="d-flex">
+                <i class="fa fa-american-sign-language-interpreting"></i>
+                <h4>Career Training</h4>
+                <p>Travel and explore the country! You’ll have plenty of time to make new friends while also gaining
+                    work experience.</p>
+            </article>
+        </div>
+        <p class="all-program-parent"><a class="all-program" href="#">see all the program</a></p>
```
Comment (henatan99) [inline index.html:125]: - [x] There should be another button element in mobile view while the `p` element with `see all the program` is for desktop view.
- [x] Again please mind spacings/paddings/margins. Make a comparison using the attached screenshot.

image
image

File: css/about.css [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+
+#ourlogo {
+  height: 60vh;
+  padding: 20px;
+}
+
+#ourlogo .container {
+  justify-content: center;
+  align-items: center;
+}
```
Comment (henatan99) [inline css/about.css:63]: - [x] Use`flex-box` to align all items in `ourlogo` section in column. 
image
image

File: css/about.css [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+  margin-top: 5px;
+}
+
+.last-activities {
+  background-color: #272a31;
+  color: white;
+  height: 80vh;
+  text-align: center;
+  font-weight: 900;
+}
```
Comment (henatan99) [inline css/about.css]: - [x] The background and font colors should comply with the original design.
image
image

File: about.html [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+            </div>
+        </div>
+    </nav>
+    <!-- End of Header -->
+    <!--     hero section -->
+    <section id="hero">
+        <div class="hero container d-flex flex-dir-col">
+            <div class="hero-title">
+                <h2>Coding & Travelling<span></span></h2>
+                <h2>Meeting <span></span></h2>
```
Comment (henatan99) [inline about.html]: - [x] You need to have at least one highest heading text `h1` tag and this one looks the right one for this page. Please do the same for the other page too.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/about.css (hunk 9 above)
Comment (Hamzaoutdoors) [reply css/about.css]: Done thanks for your review 👍

File: about.html (hunk 10 above)
Comment (Hamzaoutdoors) [reply about.html]: Done thanks for your review 👍

File: css/about.css (hunk 8 above)
Comment (Hamzaoutdoors) [reply css/about.css:63]: Done thanks for your review 👍

File: index.html (hunk 7 above)
Comment (Hamzaoutdoors) [reply index.html:125]: issue resolved thanks for your review 👍

Comment (Hamzaoutdoors) [reply index.html:125]: issue resolved thanks for your review 👍

File: css/main.css (hunk 6 above)
Comment (Hamzaoutdoors) [reply css/main.css]: issue resolved thanks for your review 👍

File: css/main.css (hunk 3 above)
Comment (Hamzaoutdoors) [reply css/main.css]: issue resolved thanks for your review 👍

File: index.html (hunk 4 above)
Comment (Hamzaoutdoors) [reply index.html]: Done thanks for your review 👍

File: about.html (hunk 1 above)
Comment (Hamzaoutdoors) [reply about.html:55]: issue resolved thanks for your review 👍

File: css/main.css (hunk 5 above)
Comment (Hamzaoutdoors) [reply css/main.css:200]: issue resolved thanks for your review 👍

File: index.html (hunk 2 above)
Comment (Hamzaoutdoors) [reply index.html:31]: issue resolved thanks for your review 👍

**Review by bellom — APPROVED** [review]
## Hi @Hamzaoutdoors  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**